client-internal/src/main/java/org/terracotta/angela/client/AngelaOrchestrator.java

```diff
@@ -48,7 +49,7 @@ private AngelaOrchestrator(boolean local) {
       PortAllocator.PortReservation reservation = portAllocator.reserve(2);
       int igniteDiscoveryPort = reservation.next();
       int igniteComPort = reservation.next();
-      this.localAgent = Agent.startCluster(Collections.singleton("localhost:" + igniteDiscoveryPort), "localhost:" + igniteDiscoveryPort, igniteDiscoveryPort, igniteComPort);
+      this.localAgent = Agent.startCluster(Collections.singleton(IpUtils.getHostName() + ":" + igniteDiscoveryPort), IpUtils.getHostName() + ":" + igniteDiscoveryPort, igniteDiscoveryPort, igniteComPort);
```

Comment:
"peers" are used to feed Ignite discovery mechanism (addresses and ports). 
So no localhost can be used.
This code was here even before Myron's refactoring, but I guess when local mode was introduced, something has been messed up. Before my last refactorings, this line was already there (in an old class called ClusterAgent)